astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+            \midrule % line separating two internal rows
+            0 & Gather documentation on all equipment and preliminary work & Summer \\
+            1 & Review existing designs and materials for roll off motor & 1 week \\
+            2 & System development of roll off motor
+            Repair existing roll off motor
+            Design roll off warning systems
+            Order parts for schematics
+            Install system at observatory & 4 weeks \\
+            3 & Review existing designs and materials for latch arm & 1 week \\
+            4 & System development of latch arm assembly
+            Get latch arm working in current state
+            Design schematic to fix power usage
+            Order parts for schematics
+            Solder modifications to latch arm controls
+            Install system at observatory & 3 weeks \\
+            5 & Review existing designs and materials for telescope modifications & 1 week \\
+            6 & System development of latch arm assembly
+            Design mounting for selected LED
+            Determine placement of modifications
+            Install system at observatory & 2 weeks \\
+            7 & Systems testing & 2 weeks \\
+            8 & Generate documentation and delivery to SPEX & 1 week \\
+            % LaTeX doesn't really like multi-line cell contents. Try to keep the text in each cell concise!
+            \bottomrule
+        \end{tabularx}
+    \label{tab:timeline2}
+    \end{table}
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+  Leadership with the prerequisite skills has been determined.
+  It would be beneficial for some students to already have basic engineering and hand tool skills.
+  Knowledge of Arduino code and basic electrical circuits is also beneficial.
+  These skills do not need to be present in all group members as these projects are an opportunity for new members to learn these skills.
+  For the roll off motor repair, someone with some motor knowledge would be crucial.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+  Primary funding for these projects will come from the Observatory itself.
+  If extra funds are needed, RIT SPEX Astrodynamics may apply for SG funding due to their newly developed SG club status.
+  \autoref{tab:costEstimate} outlines the estimated costs of the projects.
+  The costs have been overestimated for planning purposes.
+
+  \begin{table}[hb!]
+    % this table is too wide for the two-column format, so we let it expand across both columns
+    % we haven't told LaTeX where to put this so it'll find the best place.
+        \caption{Estimated Budget}
+        \centering
+        \begin{tabularx}{\linewidth}{lXl}
```

Comment:
you have 3 columns defined but only 2 columns of data. replace the tabularx with a regular tablular, you dont need word wrap.

```tex
\begin{tabular}{ll}
```
dont forget to replace the tabularx with tabular at the `\end{}` too